unstable_rsc-vite-spa/src/entry.client.tsx

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
I switched back to `entry.browser.tsx` 🙈

[context](https://github.com/remix-run/react-router/pull/13971#discussion_r2200674939)